spec/Records.tex

```diff
@@ -522,7 +434,7 @@ \subsection{Record Assignment}
 a default assignment operator for each record type \chpl{R} having the signature
 \begin{example}
 \begin{chapel}
-proc =(ref lhs:R, rhs:R) : void ;
+proc =(ref lhs:R, rhs) : void ;
 \end{chapel}
 \end{example}
 \noindent
```

Comment:
Separately, at the end of the sentence "The compiler-provided assignment operator may be overridden" let us add:

```
as described at the end of \ref{Assignment_Statements}
```